starboard/shared/starboard/player/testdata/sha1_files.gni

```diff
@@ -14,6 +14,8 @@
 
 sha1_files = [
   "beneath_the_canopy_137_avc.dmp.sha1",
+  "beneath_the_canopy_137_avc_480.dmp.sha1",
```

Comment:
The number here is an itag, so likely shouldn't be 137 for 480p.

Reply:
fixed, thanks